Content.Shared/CCVar/CCVars.cs

```diff
@@ -1046,6 +1046,15 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<float> AtmosTickRate =
             CVarDef.Create("atmos.tickrate", 15f, CVar.SERVERONLY);
 
+        /// <summary>
+        ///     Scale factor for how fast things happen in our atmosphere
+        ///     simulation compared to real life. 1x means that a room takes as
+        ///     long to heat up in game as real life. Players typically expect
+        ///     things to happen faster in-game.
+        /// </summary>
+        public static readonly CVarDef<float> AtmosSpeedup =
+            CVarDef.Create("atmos.speedup", 64f, CVar.SERVERONLY);
+
```

Comment:
The description & naming of the cvar is pretty inaccurate, it makes it sound like a time scale, which it isn't. At the very least this should just be two separate cvars (pump speed and heat capacity scale). Though I'm still not sure why heat capacity is being globally scaled.

Reply:
As explained in the PR description, heat capacity is being scaled to make heaters and freezers "faster".

The reason it is a speedup is because it modifies the [thermal time constant](https://en.wikipedia.org/wiki/Time_constant#Thermal_time_constant) which is a product involving heat capacity. Reducing heat capacity by 10× speeds up the thermal time constant by 10×. Therefore, it actually effectively a time speedup.